doc/libNX_X11/lcUniConv/8bit_tab_to_h.c

```diff
@@ -124,6 +124,7 @@ int main (int argc, char *argv[])
       strcpy(fname,directory); strcat(fname,filename);
       f = fopen(fname,"w");
       if (f == NULL)
+        free(fname);
```

Comment:
the free() is ok here, but I miss a check if the malloc() failed.

Reply:
done

Reply:
i think we could use asprintf here, much shorter code.